app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
The line `self.people[name] = self` will raise an error because `self.people` is trying to access an instance attribute that doesn't exist. You should use `Person.people[name] = self` to access the class attribute.